pkg/tests/handlers/clusters.go

```diff
@@ -55,8 +55,11 @@ func generateCreateClusterTargeter(ctx context.Context, ID, method, url string,
 		fakeClusterProps := map[string]string{
 			"fake_cluster": "true",
 		}
+		awsTags := map[string]string{
+			"User": "pocm-perf",
```

Comment:
just one suggestion, could you change the `User` tag with `CI-User` ? better avoid these common key names as it might conflict with AWS default tags.

Reply:
@krishvoor confirmed that cloud-governance team would be interested on this `User` tag only for their cleanup action. Let it be then. LGTM